src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -538,7 +569,7 @@ private IEnumerable<RuntimeLibrary> GetDirectReferenceRuntimeLibraries()
                     name: GetReferenceLibraryName(r),
                     version: r.Version,
                     hash: string.Empty,
-                    runtimeAssemblyGroups: new[] { new RuntimeAssetGroup(string.Empty, r.FileName) },
+                    runtimeAssemblyGroups: new[] { new RuntimeAssetGroup(string.Empty, new[] { CreateRuntimeFile(r.FileName, r.FullPath) }) },
```

Comment:
Note - when @tannergooding's change #2090 goes in, we will need to merge this with that, so the new references get `CreateRuntimeFile` called on them as well.

Reply:
#2090 was merged, was this handled?

Reply:
@tannergooding @eerhardt It looks like this is handled correctly, as the new code from #2090 ends up calling through `GetReferenceRuntimeLibraries` for the indirect references, the same as how the direct references are handled.